### PR TITLE
[FLINK-17976][doc] Synchronize Chinese native Kubernetes document

### PR DESCRIPTION
**Pull request by wangyang0918**
Translate the native K8s document changes in #12558 to Chinese.

### CODE REVIEW

**Comment by wangyang0918** [comment]
cc @klion26 Could you please have a look this change?

File: docs/ops/deployment/native_kubernetes.zh.md [hunk 1]
```diff
@@ -100,7 +100,7 @@ $ ./bin/flink run -d -t kubernetes-session -Dkubernetes.cluster-id=<ClusterId> e
 ### 访问 Job Manager UI
 
 有几种方法可以将服务暴露到外部（集群外部） IP 地址。
-可以使用 `kubernetes.service.exposed.type` 进行配置。
+可以使用 [`kubernetes.rest-service.exposed.type`]({% link ops/config.md %}#kubernetes-rest-service-exposed-type) 进行配置。
```
Comment (klion26) [inline docs/ops/deployment/native_kubernetes.zh.md:103]:
```suggestion
可以使用 [`kubernetes.rest-service.exposed.type`]({% link ops/config.zh.md %}#kubernetes-rest-service-exposed-type) 进行配置。
```

File: docs/ops/deployment/native_kubernetes.zh.md [hunk 2]
```diff
@@ -113,10 +113,12 @@ $ kubectl port-forward service/<ServiceName> 8081
 - `NodePort`：通过每个 Node 上的 IP 和静态端口（`NodePort`）暴露服务。`<NodeIP>:<NodePort>` 可以用来连接 JobManager 服务。`NodeIP` 可以很容易地用 Kubernetes ApiServer 地址替换。
 你可以在 kube 配置文件找到它。
 
-- `LoadBalancer`：默认值，使用云提供商的负载均衡器在外部暴露服务。
+- `LoadBalancer`：使用云提供商的负载均衡器在外部暴露服务。
 由于云提供商和 Kubernetes 需要一些时间来准备负载均衡器，因为你可能在客户端日志中获得一个 `NodePort` 的 JobManager Web 界面。
 你可以使用 `kubectl get services/<ClusterId>` 获取 EXTERNAL-IP 然后手动构建负载均衡器 JobManager Web 界面 `http://<EXTERNAL-IP>:8081`。
 
+  <span class="label label-warning">警告!</span> JobManager可能会在无需认证的情况下暴露在公网上，同时可以提交任务运行.
```
Comment (klion26) [inline docs/ops/deployment/native_kubernetes.zh.md:120]:
```suggestion
  <span class="label label-warning">警告!</span> JobManager 可能会在无需认证的情况下暴露在公网上，同时可以提交任务运行。
```

**Comment by rmetzger** [comment]
@wangyang0918 Awesome, thanks a lot for opening this PR! I forgot to open a JIRA ticket for this :(
Thanks for the review @klion26. Will merge it soon!

**Pull request merged**